<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audio Controller</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="main_sink_label">
       <property name="text">
        <string>Main Sink</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="main_sink_combo_selector"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>140</y>
      <width>341</width>
      <height>264</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QDial" name="auxillary_knob_channel_0">
       <property name="minimum">
        <number>-63</number>
       </property>
       <property name="maximum">
        <number>63</number>
       </property>
       <property name="singleStep">
        <number>4</number>
       </property>
       <property name="pageStep">
        <number>16</number>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="notchTarget">
        <double>6.700000000000000</double>
       </property>
       <property name="notchesVisible">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDial" name="auxillary_knob_channel_1">
       <property name="minimum">
        <number>-63</number>
       </property>
       <property name="maximum">
        <number>63</number>
       </property>
       <property name="singleStep">
        <number>4</number>
       </property>
       <property name="pageStep">
        <number>16</number>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="notchTarget">
        <double>6.700000000000000</double>
       </property>
       <property name="notchesVisible">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QDial" name="auxillary_knob_channel_2">
       <property name="minimum">
        <number>-63</number>
       </property>
       <property name="maximum">
        <number>63</number>
       </property>
       <property name="singleStep">
        <number>4</number>
       </property>
       <property name="pageStep">
        <number>16</number>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="notchTarget">
        <double>6.700000000000000</double>
       </property>
       <property name="notchesVisible">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDial" name="auxillary_knob_channel_3">
       <property name="minimum">
        <number>-63</number>
       </property>
       <property name="maximum">
        <number>63</number>
       </property>
       <property name="singleStep">
        <number>4</number>
       </property>
       <property name="pageStep">
        <number>16</number>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="notchTarget">
        <double>6.700000000000000</double>
       </property>
       <property name="notchesVisible">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>641</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="3" column="2">
      <widget class="QLabel" name="label_channel_2">
       <property name="text">
        <string>Music</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="mute_button_channel_3">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Mute</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="mute_button_channel_4">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Mute</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="label_channel_4">
       <property name="text">
        <string>Auxillary</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="mute_button_channel_1">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Mute</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="mute_button_channel_2">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Mute</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="mute_button_channel_0">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Mute</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_channel_0">
       <property name="text">
        <string>Microphone</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_channel_3">
       <property name="text">
        <string>System</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_channel_1">
       <property name="text">
        <string>Communication</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="select_button_channel_0">
       <property name="text">
        <string>Select</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="select_button_channel_1">
       <property name="text">
        <string>Select</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="select_button_channel_2">
       <property name="text">
        <string>Select</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="select_button_channel_3">
       <property name="text">
        <string>Select</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="select_button_channel_4">
       <property name="text">
        <string>Select</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSlider" name="slider_volume_channel_0">
       <property name="maximum">
        <number>127</number>
       </property>
       <property name="pageStep">
        <number>8</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>7</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="slider_volume_channel_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="cursor">
        <cursorShape>SizeVerCursor</cursorShape>
       </property>
       <property name="maximum">
        <number>127</number>
       </property>
       <property name="pageStep">
        <number>8</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>7</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSlider" name="slider_volume_channel_2">
       <property name="maximum">
        <number>127</number>
       </property>
       <property name="pageStep">
        <number>8</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>7</number>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSlider" name="slider_volume_channel_3">
       <property name="maximum">
        <number>127</number>
       </property>
       <property name="pageStep">
        <number>8</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>7</number>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QSlider" name="slider_volume_channel_4">
       <property name="maximum">
        <number>127</number>
       </property>
       <property name="pageStep">
        <number>8</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>7</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="restart_engine_button">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>20</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Restart Engine</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
